src/components/com_tjdashboard/administrator/views/dashboard/tmpl/default.php

```diff
@@ -14,7 +14,8 @@
 
 $document->addScript(Juri::root() . 'plugins/tjdashboardrenderer/tabulator/assets/js/jquery-ui.min.js');
 $document->addStyleSheet(JUri::root() . '/media/techjoomla_strapper/bs3/css/bootstrap.css');
-$document->addStyleSheet(JUri::root() . 'media/com_tjdashboard/css/font-awesome.min.css');
+$document->addStyleSheet(JUri::root() . 'media/com_tjlms/font-awesome/css/font-awesome.min.css');
+$document->addStyleSheet(JUri::root() . 'media/com_tjlms/css/tjlms_backend.css');
```

Comment:
@monikaatvowel Why we have tjlms dependency here?

Reply:
@monikaatvowel  you will need to add the condition in the onAfterRoute function of tjlms system plugin. We alredy have code for jlike and categories written there

Reply:
done. removed from here and added there